src/SOS/Strike/strike.cpp

```diff
@@ -200,6 +200,12 @@ HMODULE g_hInstance = NULL;
 #define IfFailRet(EXPR) do { Status = (EXPR); if(FAILED(Status)) { return (Status); } } while (0)
 #endif
 
+bool IsWindowsTarget()
```

Comment:
This seems like it should be in runtime.cpp with an extern in runtime.h.  It may be useful in files other than strike.cpp.

Reply:
Yeah I was originally going to put it there...  I guess I got lazy.  I fix it.